### PR TITLE
improve the documentation of the action

### LINKED ISSUE #20: Link to example issues from readme ?
Thanks for your comment on https://github.com/scientific-python/specs/issues/181, Looking at the readme I'd love to see the kind of issue it opens !

### PR DESCRIPTION
**Pull request by keewis**
So far the documentation was pretty minimal: a simple example, no explanation of the options, no real-world examples.

- [x] closes #20

### CODE REVIEW

**Comment by keewis** [comment]
cc @jrbourbeau, I hope you don't mind me using `dask` as an example

File: README.md [hunk 1]
```diff
@@ -18,15 +26,64 @@ jobs:
...
         failure()
         && ...
       with:
         log-path: pytest-log.jsonl
 ```
+
+See [this repository](https://github.com/keewis/reportlog-test/issues) for example issues. For more realistic examples, see
+
+- `xarray` ([workflow](https://github.com/pydata/xarray/blob/main/.github/workflows/upstream-dev-ci.yaml), [example issue](https://github.com/pydata/xarray/issues/6197))
+- `dask` ([workflow](https://github.com/dask/dask/blob/main/.github/workflows/upstream.yml), [example issue](https://github.com/dask/dask/issues/10347))
```
Comment (jrbourbeau) [inline README.md:55]:
```suggestion
- `dask` ([workflow](https://github.com/dask/dask/blob/main/.github/workflows/upstream.yml), [example issue](https://github.com/dask/dask/issues/10089))
```

**Review by jrbourbeau — COMMENTED** [review]
No problem at all! Only suggestion would be to point to an already closed issue : )

**Comment by jrbourbeau** [comment]
Btw, I wrote a short blogpost about how we're using this over in the Dask / Coiled blogs that give you a shoutout https://blog.dask.org/2023/04/18/dask-upstream-testing. I can't remember if I sent you this already or not -- hope that's okay

File: README.md (hunk 1 above)
Comment (keewis) [reply README.md]: of course!

**Comment by keewis** [comment]
I don't remember whether you pinged me regarding that, but I do remember reading that blog post and being happy that the action is useful for other people. In other words, yes, that's totally okay!

**Pull request merged**